packages/navigator/src/browser/navigator-filter.ts

```diff
@@ -49,7 +49,7 @@ export class FileNavigatorFilter {
         let hasChanged = false;
         const { preferenceName, newValue } = event;
         if (preferenceName === 'navigator.exclude') {
-            this.filterPredicate = this.createFilterPredicate(newValue || {});
+            this.filterPredicate = this.createFilterPredicate(newValue as FileNavigatorFilter.Exclusions | undefined || {});
```

Comment:
Did it cause a problem without the explicit cast?

Reply:
Yes, because `newValue` is of union type of all the preferences.

Reply:
Thank you for your explanation.